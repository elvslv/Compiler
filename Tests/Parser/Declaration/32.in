type a = array[1..1] of integer;
     b = array[1..1] of a;
     c = array[1..1] of b;