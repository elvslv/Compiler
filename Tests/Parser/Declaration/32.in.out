type A = array [const 1..const 1] of INTEGER;
type B = array [const 1..const 1] of A;
type C = array [const 1..const 1] of B;
